### PR TITLE
Add SQL region annotation examples

### PR DESCRIPTION
**Pull request by adeverteuil**
**What this PR does / why we need it**:

Add SQL example for region annotation queries in MySQL, PostgreSQL and MSSQL.

Also specifies this is supported from Grafana v6.6+.

I didn't test the rendered HTML output in a dev environment.

### CODE REVIEW

**Review by oddlittlebird — COMMENTED** [review]
LGTM, but I suggest getting a technical review from someone who knows SQL.

File: docs/sources/features/datasources/mysql.md [hunk 1]
```diff
@@ -350,6 +350,22 @@ WHERE
   $__unixEpochFilter(epoch_time)
 ```
 
+**Example region query using time and timeend columns with epoch values:**
+
+From Grafana v6.6, region annotations (with a `timeend`) are supported in the MySQL plugin.
```
Comment (marefr) [inline docs/sources/features/datasources/mysql.md:355]:
```suggestion
> Only available in Grafana v6.6+.
```

File: docs/sources/features/datasources/postgres.md [hunk 2]
```diff
@@ -354,6 +354,22 @@ WHERE
   $__unixEpochFilter(epoch_time)
 ```
 
+**Example region query using time and timeend columns with epoch values:**
+
+From Grafana v6.6, region annotations (with a `timeend`) are supported in the PostgreSQL plugin.
```
Comment (marefr) [inline docs/sources/features/datasources/postgres.md:359]:
```suggestion
> Only available in Grafana v6.6+.
```

**Review by marefr — CHANGES_REQUESTED** [review]
Some suggestions to follow how we mark section to be supported in certain Grafana versions. Otherwise looks good

**Review by marefr — APPROVED** [review]
Applied my suggestions. @oddlittlebird please approve/merge if you agree

**Review by aknuds1 — APPROVED** [review]
LGTM

**Comment by oddlittlebird** [comment]
LGTM.

**Comment by marefr** [comment]
Thanks @adeverteuil

**Pull request merged**